src/bcrhp/bcrhp/migrations/909_update_url_prefix.py

```diff
@@ -0,0 +1,542 @@
+# Generated by Django 4.2.13 on 2024-07-05 09:19
+
+from django.db import migrations
+from bcrhp.migrations.operations.privileged_sql import RunPrivilegedSQL
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bcrhp', '947_historic_site_one_line'),
+    ]
+
+    operations = [
+        RunPrivilegedSQL(
+            """
+            begin;
+            update map_layers
+            set legend = replace(legend, '/int/bcrhp/', '/bcrhp/');
+            commit;
+            """,
+            """
+            begin;
+            update map_layers
+            set legend = replace(legend, '/bcrhp/', '/int/bcrhp/');
+            commit;
+            """
+        ),
+        RunPrivilegedSQL(
+            """
+            begin;
+            update map_sources
```

Comment:
This query nulls out map sources: mapbox-satellite, search-query and geocode-point values. Needs to be limited to rows it should change.

Reply:
Updated